### ISSUE TITLE
Examine verb lets you examine all your mob contents, including organs

### ISSUE THREAD

**Issue by Ashturian** [Eclipse-Station/NEV-Northern-Light#302]
Accessed specifically by using "examine" from the hotbar and bringing up the dialogue to choose an object in view.

**Comment by ShadzFen** [comment]
Looks like this issue is fixed. I examined myself and another mob using the Examine verb in the IC menu, nothing about organs anything like that.

![image](https://user-images.githubusercontent.com/45645502/87339346-c2837880-c546-11ea-888a-ca1be9e5dcfc.png)

**Comment by Ashturian** [comment]
![examine](https://user-images.githubusercontent.com/53239656/87778687-1b8b2f00-c81b-11ea-9408-bc3626ac03bc.png)
Ah, it's not through examining things the normal way, ie right click or shift+left click, this is by using "Examine" in the console or the IC verbs tab

**Comment by ShadzFen** [comment]
Yeah I used the IC verbs tab last time for that pic I posted. Strange... I'll give it another look at some point

**Comment by ShadzFen** [comment]
Alright I understand what you mean now. It's not that the body parts show up when examining someone, it's that the body parts CAN be examined individually on the context menu. I'm just a little thick, do excuse me.

![image](https://user-images.githubusercontent.com/45645502/87951243-a0d74380-caa8-11ea-814b-ca46114e886d.png)
